app/src/main/res/values/colors.xml

```diff
@@ -33,4 +33,6 @@
 
   <color name="search_query_highlight">#52FFC800</color>
 
+  <color name="new_added_identifier_highlight">#FFF8E0</color>
```

Comment:
We already have this color as `simple_yellow_100` in the colors.xml, can we use that instead?

Reply:
I've added this as a new commit @Honey14